Gemfile.lock

```diff
@@ -78,7 +85,7 @@ PATH
       claide (~> 0.9.1)
       cocoapods-core (= 0.39.0.beta.3)
       cocoapods-downloader (~> 0.9.3)
-      cocoapods-plugins (~> 0.4.2)
```

Comment:
is this an accident? I still see the reference below?

`   s.add_runtime_dependency 'cocoapods-plugins',    '~> 0.4.2'`